runtime/js-compute-runtime/builtins/cache-simple.cpp

```diff
@@ -346,29 +296,25 @@ bool SimpleCache::getOrSetThenHandler(JSContext *cx, JS::HandleObject owner, JS:
     if (!JS_HasProperty(cx, insertionObject, "length", &found)) {
       return false;
     }
-    if (!found) {
-      JS_ReportErrorASCII(cx, "SimpleCache.getOrSet: length property is required when the value "
```

Comment:
The length parameter is not actually required, what happens if the length is supplied is that any clients that begin streaming the item’s contents before it is completely provided will see the promised length which allows them to, for example, use content-length instead of transfer-encoding: chunked if the item is used as the body of a Request or Response